__tests__/component-options.test.js

```diff
@@ -7,11 +7,11 @@ const PORT = configPaths.testPort
 let page
 let baseUrl = 'http://localhost:' + PORT
 
-beforeAll(async () => {
+beforeEach(async () => {
   page = await setupPage()
 })
 
-afterAll(async () => {
+afterEach(async () => {
```

Comment:
Puppeteer does some clever stuff where it recycles pages so we need to ensure they're full closed and open to test the hash functionality